src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        LocalDateTime orderDate = LocalDateTime.now();
+        Order order = new Order(shoppingCart.getUser(), orderDate);
+        order.setTickets(new ArrayList<>(shoppingCart.getTickets()));
```

Comment:
I prefer to initialize all fields by the constructor :  
Order order = new Order(shoppingCart.getUser(), LocalDateTime.now(), new ArrayList<>(shoppingCart.getTickets()));
or use only setters.